tests/phpunit/Integration/Query/Fixtures/format-02-02-table-output-en.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
We have some test code somewhere that will automatically find this file and use it?